src/utility/watchdog/Watchdog.h

```diff
@@ -18,21 +18,6 @@
 #ifndef ARDUINO_AIOTC_UTILITY_WATCHDOG_H_
 #define ARDUINO_AIOTC_UTILITY_WATCHDOG_H_
 
-/******************************************************************************
- * INCLUDE
- ******************************************************************************/
-
-#include <AIoTC_Config.h>
-
-/******************************************************************************
- * PREPROCESSOR SECTION
- ******************************************************************************/
-
-#ifdef ARDUINO_ARCH_SAMD
-#  include <Adafruit_SleepyDog.h>
```

Comment:
Is sleepydog included somewhere else?

Reply:
No, that's the only spot. But this isn't a change particular to this PR, I simply moved the inclusion from .h to .cpp.